STYLE_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+# Go Coding Style Guide
+
+In order to keep our code looking good with lots of programmers working on it, it helps to have a "style guide", so all
+the code generally looks quite similar. This doesn't mean there is only one "right way" to write code, or even that this
+standard is better than your style.  But if we agree to a number of stylistic practices, it makes it much easier to read
+and modify new code. Please feel free to make suggestions if there's something you would like to add or modify.
+
+Please, before going through the rest of the document, have a read over the [syntactical style](https://github.com/uber-go/guide/blob/master/style.md)
+
+If you wish to gain some further understanding, we recommend having a read on how to code [effective go](https://golang.org/doc/effective_go.html)
```

Comment:
This is the required reading, IMO

Reply:
I might make these two paragraphs something like: 

```md
We expect all contributors to be familiar with [Effective Go](https://golang.org/doc/effective_go.html) (and it's recommended reading for all Go programmers anyways.) Additionally, we generally agree with the suggestions in [Uber's style guide](https://github.com/uber-go/guide/blob/master/style.md) and use that as a starting point.
```

Reply:
Yup I agree, that sounds better